src/guiguts/utilities.py

```diff
@@ -38,3 +44,21 @@ def _is_system(system: str) -> bool:
         if _is_system.system not in ["Darwin", "Linux", "Windows"]:
             raise Exception("Unknown windowing system")
         return _is_system.system == system
+
+
+def load_dict_from_json(filename: str) -> Optional[dict[str, Any]]:
+    """If file exists, attempt to load into dict.
+
+    Args:
+        filename: Name of JSON file to load.
+
+    Returns:
+        Dictionary if loaded successfully, or None.
+    """
+    if os.path.isfile(filename):
+        with open(filename, "r") as fp:
+            try:
+                return json.load(fp)
+            except json.decoder.JSONDecodeError as exc:
+                logger.error(f"Unable to load {filename}\n" + str(exc))
```

Comment:
Consider `Unable to parse file {filename}` or `Unable to load {filename} -- not valid JSON format` or something similar to indicate to the user that the problem wasn't with _accessing_ the file (doesn't exist, can't read it) but with the parsing (exists and readable but not formatted correctly).

Reply:
Thanks - that would be clearer. The Exception which is output on the line below gives details, e.g. "Line 3, column 4: expected colon" so if they have edited the file, there's a chance they will be able to correct it.